src/Entity/ClickNCollectShippingMethod.php

```diff
@@ -22,16 +22,10 @@
  */
 trait ClickNCollectShippingMethod
 {
-    /**
-     * @var Collection|LocationInterface[]
```

Comment:
You should keep this for auto-completion, static analysis, etc. This can be: `Collection<LocationInterface>`